mappings/net/minecraft/client/texture/PlayerSkinProvider.mapping

```diff
@@ -26,3 +33,6 @@ CLASS net/minecraft/unmapped/C_pyciilvf net/minecraft/client/texture/PlayerSkinP
 			ARG 1 type
 			ARG 2 id
 			ARG 3 texture
+	CLASS C_wbpmwuvo
```

Comment:
could you name this class please?

Reply:
It's a synthetic class. But this will cause problems if not named...

Reply:
mhm, I was thinking we'd want to avoid filling it with hashed